omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -238,6 +238,7 @@
 # See default['private_chef']['postgresql']['db_connection_superuser'] for information
 default['private_chef']['opscode-erchef']['sql_connection_user'] = nil
 default['private_chef']['opscode-erchef']['enable_request_logging'] = true
+default['private_chef']['opscode-erchef']['disallow_non_manage_email_update'] = false
```

Comment:
NIT: something to the effect allow_email_update_only_from_manage may read better?